Blockzilla/metrics.yaml

```diff
@@ -554,7 +554,9 @@ browser_search:
       the name of the search engine. If it's a custom search engine (defined:
       https://github.com/mozilla-mobile/fenix/issues/1607) the value will be
       `custom`.
-      `source` will be: `action`, `suggestion`.
+      `source` will be: `action`, `suggestion`, `widget`, `shortcut`, `topsite`
```

Comment:
I'm not the right person to review this, but please make sure that the metric implementation matches what Fenix does.

Reply:
Okay. Yes, the implementation matches Fenix.

Reply:
Thanks! Can't wait to see this landed :-)